compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -296,6 +296,32 @@ impl IntrinsicCallMethods<'tcx> for Builder<'a, 'll, 'tcx> {
                 }
             }
 
+            sym::raw_eq => {
```

Comment:
This will need a cg_clif impl.

Reply:
I figured, but I couldn't figure out where to put a test for such, nor how to test cranelift via x.py.  (Looking for [cranelift in the dev guide](https://rustc-dev-guide.rust-lang.org/?search=cranelift) didn't really help.)

Can you point me to some instructions for this?  Ideally ones that don't need `.sh` scripts, since my rust dev box is Windows...

Reply:
You could add `cranelift` to the `rust.codegen-backends` array in `cargo.toml` build rustc and then compile a test hitting this intrinsic using `-Zcodegen-backend=cranelift`. It is possible that you may need to cherry-pick the bootstrap changes of https://github.com/rust-lang/rust/pull/81746, but I think using `rustup toolchain link` works fine even without. I think only `./x.py dist` and `./x.py install` are broken for cg_clif on master.

Reply:
Thanks!  Turns out I'd already done all those steps except for using the `-Z` flag 🤦

Just got the "unsupported intrinsic" error; I'll get right on fixing it.